modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,102 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups are also referred to as organizations (or sub organizations). Business groups reside within the master organization and can contain their own business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
+
+[NOTE]
+Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative. There is a limit of 100 business groups per organization.
+
+If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. The organization administrator is also the administrator of any business groups (sub organizations) within the parent organization.
+
+[NOTE]
+When you create a new business group, all the users with the *Organization Administrators* role in the parent business group are also added as organization administrators in the newly created business group (the child business group). However, when you create a new user as an *Organization Administrator* in the parent business group, the user is not added to all the sub business groups.
+
+=== Create a Business Group
+
+Create business groups within your organization. 
+
+[NOTE]
+There is a limit of 100 business groups per organization. 
```

Comment:
Instead of burying this limit on the task topic in a note, maybe add to conceptual topic and give it a heading that includes Limit?